src/Admin/Menu/MenuNodeAdmin.php

```diff
@@ -69,14 +70,24 @@ protected function configureFormFields(FormMapper $formMapper)
                                 'content' => 'content',
                             ),
                             'map' => array(
-                                'route' => array('link'),
+                                'route' => array('link','routeParameters'),
                                 'uri' => array('link'),
                                 'content' => array('content', TreeSelectType::class),
                             ),
                             'placeholder' => 'auto',
                             'required' => false,
                         ))
                         ->add('link', TextType::class, array('required' => false, 'mapped' => false))
+                        ->add('routeParameters', KeyValueType::class, array(
```

Comment:
Thansk for you feedback. I'll do it, just need time. 
Question... shall we filter only route parameters or whole route configuration options?